app/navigation/NavigatorForCorrectReturn.scala

```diff
@@ -28,11 +27,13 @@ import javax.inject.{Inject, Singleton}
 class NavigatorForCorrectReturn @Inject()() extends Navigator {
 
   override val normalRoutes: Page => UserAnswers => Call = {
+    case OperatePackagingSiteOwnBrandsPage => userAnswers => defaultCall
```

Comment:
my understanding is that scaffold sets up navigation for it is automatically go from the question page to the litres page if answer is yes - this is not currently working for these pages